components/match2/commons/match_group_display_helper.lua

```diff
@@ -154,6 +141,26 @@ function DisplayHelper.MapAndStatus(game, config)
 	return mapText .. (statusText or '')
 end
 
+---Displays the map name and link.
+---@param game MatchGroupUtilGame
+---@param config {noLink: boolean?}?
+---@return string
+function DisplayHelper.Map(game, config)
+	config = config or {}
+	local mapText
+	if game.map and game.mapDisplayName then
+		mapText = '[[' .. game.map .. '|' .. game.mapDisplayName .. ']]'
+	elseif game.map and not config.noLink then
+		mapText = '[[' .. game.map .. ']]'
```

Comment:
possibly use Module:Page?

Reply:
I'll leave it out of scope, just extracted a part of a function to a different function